graphene_django/utils/testing.py

```diff
@@ -40,8 +40,13 @@ def query(self, query, op_name=None, input_data=None):
         body = {"query": query}
         if op_name:
             body["operation_name"] = op_name
+        if variables:
+            body["variables"] = variables
         if input_data:
-            body["variables"] = {"input": input_data}
+            if variables in body and isinstance(body, dict):
```

Comment:
Why do you need the `isinstance` check here? `body` will always be a dict because it's defined that way above.

Reply:
I actually meant to check `isinstance(body['variables'], dict)` (just to make sure that the user actually provided a dict to that argument).  but I removed it, because it's a) already specified in the docs, b) not checked for `input_data`  and c) there isn't any fall-back error handling anyway.